autocertkit/network_tests.py

```diff
@@ -267,31 +267,22 @@ def run_iperf_server(self):
                 test_ip = self.vm_info[self.server]['ip_t']
                 protocol = ''
 
-            cmd_str = "iperf -s -D %s -B %s < /dev/null >&/dev/null" \
+            cmd_str = "iperf3 -s -D %s -B %s < /dev/null >&/dev/null" \
                       % (protocol, test_ip)
             ssh_command(mip, self.username, self.password, cmd_str)
 
     def parse_iperf_line(self, data):
         """Take a CSV line from iperf, parse, returning a dictionary"""
-        lines = data.strip().split('\n')
-        log.debug("Iperf Lines: %s" % lines)
-        arr = lines[0].split(',')
+        log.debug("Iperf data: %s" % data)
+        arr = json.loads(data)
         rec = {}
-        rec['datetime'] = arr[0]
-        rec['client_ip'] = arr[1]
-        rec['client_port'] = arr[2]
-        rec['server_ip'] = arr[3]
-        rec['server_port'] = arr[4]
-        rec['id'] = arr[5]
-        rec['interval'] = arr[6]
-        rec['transfer'] = arr[7]
-        rec['bandwidth'] = arr[8]
```

Comment:
`bandwidth` is useful, should keep it and I saw iperf3 still support it from https://iperf.fr/iperf-doc.php